recipes/loro/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You can remove this file since the license file is included in the source distribution